backup/clone-node.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -euo pipefail
+
+
+clone_basedir="/opt/clone"
+remote_backup_basedir="/opt/backup"
+
+clone_lockdir="${clone_basedir}/.lock"
+
+if ! mkdir "${clone_lockdir}"; then
+    echo "Failed to acquire lock dir '${clone_lockdir}' - did a previous clone attempt fail?"
+    exit 1
+fi
+
+remote_address="$(cat "$clone_basedir/source_node")"
+
+remote="ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -i ${clone_basedir}/.ssh/backup_key ${remote_address} sudo"
```

Comment:
It could be an array like in other script, but it's up to you. This would make sure that `${remote_address}` is treated as a single string during invocation, if for some weird reason `$clone_basedir/source_node` has a multiword stuff in it.

Reply:
Yupp, makes a lot of sense. Updated.